wordpress-mobile/WordPress-iOS
# Remove Giphy dependency

Giphy recently updated their SDK, which now requires opting into use an advertising identifier and serve advertisements. We don't want to do this, so this PR removes Giphy as a user-facing Gif import option and our dependency on the Giphy SDK.

> We serve sponsored content every now and then so we can continue building great products and sharing the joy of GIFs across the Internet, and in apps like yours.
> 
> As a result, you’ll have to check a few boxes when you submit your app for review on the App Store if you’ve integrated this SDK.
> 
>     Answer yes when asked if the app uses an advertising identifier and be sure to check:
>         - Serve advertisements within the app
>         - Attribute an action taken within this app to a previously served advertisement

In a future PR we'll either look at removing the rest of the integration or repurposing it for a different gif provider.

**To test:**

* Rake dependencies
* Build and run, check it builds okay
* In the Media Library, tap + and ensure Giphy doesn't show as an option
* In Aztec, tap the `+` and then `...` on the format bar and ensure Giphy doesn't show as an option

Update release notes:
